src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
 export class Triangle {
```

Comment:
all figures must implement class Figure

Reply:
Thanks for the answer. Tried to fix this problem.